design-documents/graph-ql/coverage/returns.graphqls

```diff
@@ -0,0 +1,197 @@
+type Query{
+    # See https://github.com/magento/architecture/blob/master/design-documents/graph-ql/framework/attributes-metadata.md
+    pageSpecificCustomAttributes(
+        page_type: CustomAttributesPageEnum!
+    ): CustomAttributeMetadata
+}
+
+type Mutation {
+    createReturn(input: CreateReturnInput!): CreateReturnOutput @doc(description: "Create a new return.")
+    addReturnComment(input: AddReturnCommentInput!): AddReturnCommentOutput @doc(description: "Add a comment to an existing return.")
+    addReturnTracking(input: AddReturnTrackingInput!): AddReturnTrackingOutput @doc(description: "Add tracking information to the return.")
+    removeReturnTracking(input: RemoveReturnTrackingInput!): RemoveReturnTrackingOutput @doc(description: "Remove tracking information from the return.")
+}
+
+input CreateReturnInput {
+    items: [CreateReturnItemInput]!
+    comment_text: String
+}
+
+input CreateReturnItemInput {
+    order_item_id: ID!
+    quantity_to_return: Float!
+    selected_custom_attributes: [ID!]
```

Comment:
Are `selected_custom_attributes` and `entered_custom_attributes` both used to represent the custom fields that a merchant can add to the item return interface?

![image](https://user-images.githubusercontent.com/5233399/88083772-0e3ab100-cb49-11ea-8407-a59d2c1c4577.png)


If so, it might be nice to be a bit more specific here in the naming. It took me awhile to figure out the purpose of these input fields

Reply:
Yes, these two fields represent EAV attributes of the return item.
We should have common strategy for EAV attributes here, in Customer (not supported yet in GraphQL), Customer address. Plus in product and categories we name them `custom_attributes` in the output, however there are no custom attributes which customer can input.

Not sure how to improve it, please provide concrete suggestion.